styles/style.css

```diff
@@ -0,0 +1,527 @@
+/******* Body *******/
+body {
+  margin: 0;
+  background-color: #ff9860;
+  background-image: linear-gradient(0deg, #ff9860 0%, #f3d692 38%, #fdefcd 100%);
+  font-family: 'Roboto Mono', monospace;
+}
+
+/******* Header *******/
+header {
+  background-image: url(https://images.unsplash.com/photo-1444212477490-ca407925329e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2000&q=80);
+  height: 700px;
+  background-position: center;
+  background-size: cover;
+  position: relative;
+}
+
+.video {
+  display: block;
+  position: relative;
+  width: auto;
+  min-width: 100%;
+  max-width: 100%;
+  height: auto;
+  min-height: 100%;
+  background-size: contain;
+}
+
+.video-container {
+  width: 100%;
+  height: 100%;
+  max-height: 100%;
+  max-width: 100%;
+  overflow: hidden;
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+.header-content {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  right: 0;
+  margin: 0 auto;
+  text-align: center;
+}
+
+nav {
+  display: flex;
+  justify-content: flex-end;
+}
+
+nav a:link {
+  text-decoration: none;
+  color: #FDEFCD; 
+  font-size: 23px;
+  font-weight: bold;
+  padding: 20px;
+}
+
+nav a:visited {
+  color: #F78C51;
+}
+
+nav a:hover {
+  text-decoration: underline;
+}
+
+.home img {
+  height: 40px;
+  width: 40px;
+}
+
+nav a.home {
+  padding: 10px;
+}
+
+nav a.home:hover {
+  opacity: 0.5;
+}
+
+.header-content h1 {
+  color: #FDEFCD;
+  font-family: 'Fredericka the Great', cursive;
+  font-size: 60px;
+  padding-left: 50px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+/******* Main *******/
+
+main {
+  display: grid;
+  grid-template-columns: 10% 36% 8% 36% 10%;
+  grid-template-rows: auto;
+  grid-template-areas: 
+    ". item-a item-a item-a ."
+    ". item-b . item-c .";
+  justify-items: center;
+}
+
+/*______ Accordion Dogs ______*/
```

Comment:
Ohh here's another way. What's is the different in your file between a section that has comment `/******* Header *******/` and `/*______ Accordion Dogs ______*/`?